### DISCUSSION TITLE
Please add support for Elegoo Neptune 3 Series 3D Printers

### DISCUSSION THREAD

**Discussion by 1223745302** [OrcaSlicer/OrcaSlicer#488] [Ideas]
Please add support for Elegoo Neptune 3 Series 3D Printers,I tried to modify the parameters of bumbu X1 to achieve Neptune 3 support, but this doesn't seem to work.

**Comment by Heliscone** [comment]
This would be immensely valuable to me!

**Reply by dguisti** [reply]
Same here, PrusaSlicer is fairly functional, but it is lacking in QOL features and good profile management. Neptune printers are also becoming a quite common low-budget option, and it'd be great to have a port for it.

**Reply by hanzi2u** [reply]
i myself would love to have support for my Elegoo printer! !
